nengo_viz/static/viz_sim_control.js

```diff
@@ -182,23 +173,6 @@ VIZ.TimeSlider = function(args) {
     /** scale to convert time to x value (in pixels) */
     this.kept_scale = d3.scale.linear();
     
-    /** create the overall div */
-    this.div = document.createElement('div');
```

Comment:
It seems weird that we were making new divs every time we resized. Was this a bug that was fixed by moving everything to HTML?